src/sentry/api/endpoints/source_map_debug.py

```diff
@@ -79,27 +79,38 @@ def get(self, request: Request, project: Project, event_id: str) -> Response:
         if not frame_idx:
             raise ParseError(detail="Query parameter 'frame_idx' is required")
 
+        frame_idx = int(frame_idx)
```

Comment:
Not sure how likely it is to happen, but could this raise a `ValueError` if `frame_idx` isn't a coercible int, e.g. `int('hi')`.

Reply:
i thought of it, but in https://github.com/getsentry/sentry/blob/master/src/sentry/apidocs/parameters.py#L180-L185 we define the 2 idx's as int's. not sure how strict that typing is though